src/seedu/addressbook/data/person/Person.java

```diff
@@ -16,6 +16,11 @@
     private Address address;
 
     private final UniqueTagList tags;
+    
+    private int sequenceNumber;
+    
+    private static int nextSequenceNumber=0;
```

Comment:
it may make more sense from a code readability point of view to initialize it to 1.